### ISSUE TITLE
Links Missing in Maps

### ISSUE THREAD

**Issue by EliCylogy** [fargeo/baaqmd-maps#34]
We have two bugs:

1.The links are missing from the descriptions in the maps
2.The paragraph formatting isn’t coming through. 

The missing link issue is the most critical of the two bugs.

The content is in sitecore and is pulling from:

/sitecore/content/DotGov/Home/Utils/Interactive Map Content/About Facility

See the missing format and links not working here:

![image](https://user-images.githubusercontent.com/69152305/89184670-25c16300-d567-11ea-918b-2b7f30f6348c.png)

**Note from our developer after review:**

"The new maps requests this item correctly and receives rich HTML with the links inside:

![image](https://user-images.githubusercontent.com/69152305/89184413-c19e9f00-d566-11ea-81d9-8d3fafc5c344.png)

It looks like at some point during rendering they were cut and only text is left. 

This is on maps developer side: issue should be addressed to Farallon."

Please let me know if you have any questions.

**Comment by EliCylogy** [comment]
Note this is happening for all info areas:

![image002 (10) (1)](https://user-images.githubusercontent.com/69152305/89185482-49d17400-d568-11ea-97d0-cf293efe2c49.jpg)

![image004 (9) (1)](https://user-images.githubusercontent.com/69152305/89185484-4b9b3780-d568-11ea-9fc5-81a8fde77b97.jpg)

![image004 (8) (1)](https://user-images.githubusercontent.com/69152305/89185486-4ccc6480-d568-11ea-917f-7a364e5cfd0b.jpg)




The preview link is: http://prod-cm-baaqmd-preview.cylogy.com/Dev-Testing/Maps-New
username: cylogy
password: dev@1234

**Comment by robgaston** [comment]
@EliCylogy good catch!  this was a bit of regression caused by some fixes we applied earlier this year to better support IE 11.  I've just merged a fix in for this here: https://github.com/fargeo/baaqmd-maps/commit/5273c305f5b5cfd852a9f0731957ab10b6e39b6e

You should be able to now pull the latest to address this bug; you can see the fix in action [here](https://baaqmd.fargeo-dev.com/)

**Comment by Pujapuppy** [comment]
@robgaston Hi Rob - Can you replace styles.scss with this file (and review and merge)? I've moved the style adjustments from baaqmd-maps.css to styles.scss so they won't be overwritten. The new styles.scss is here: https://drive.google.com/drive/folders/1v7w5JXgdnD4ilb9OWvXFP1u2ce3xLEzK?usp=sharing

**Comment by EliCylogy** [comment]
Hi @robgaston Can you take a look at Puja's comment above. We need to include the new styles so that we can continue testing today. Can you take a look at this today if possible?

**Comment by robgaston** [comment]
@Pujapuppy @EliCylogy I have updated the styles with your latest from google drive.  Please let me know if you need anything else!

**Comment by Pujapuppy** [comment]
@robgaston Hi Rob, I adjusted a few more styles - Can you replace styles.scss with this file (and review and merge)? https://drive.google.com/drive/folders/1v7w5JXgdnD4ilb9OWvXFP1u2ce3xLEzK?usp=sharing

**Comment by robgaston** [comment]
@Pujapuppy @EliCylogy I just updated the repo with the latest supplied by @Pujapuppy above.  Let me know if you need anything else!

**Comment by Pujapuppy** [comment]
Hi Rob - we found one more issue. I've updated the stylesheet (again). Can you replace styles.css with the new file (and review and merge)?
https://drive.google.com/drive/folders/1v7w5JXgdnD4ilb9OWvXFP1u2ce3xLEzK?usp=sharing

**Comment by robgaston** [comment]
@Pujapuppy @EliCylogy I updated the repo with the latest from google drive - let me know if you need anything else!

**Comment by EliCylogy** [comment]
Thanks @robgaston
